packages/api/src/validators/shared.ts

```diff
@@ -2,9 +2,15 @@ import Joi from "joi";
 
 export function validateBodyFromAuthentication(
   data: unknown
-): asserts data is { emailFromAuthToken: string } {
+): asserts data is {
+  emailFromAuthToken: string;
+  userSubjectFromAuthToken: string;
+} {
   const validation = Joi.object()
-    .keys({ emailFromAuthToken: Joi.string().email().required() })
+    .keys({
+      emailFromAuthToken: Joi.string().email().required(),
+      userSubjectFromAuthToken: Joi.string().required(),
```

Comment:
Joi has a "uuid" subtype of string that we should use, I think (though of course that means changing the test strings elsewhere).